scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -246,8 +263,129 @@ public SchedulerDBManager(Configuration configuration, boolean drop) {
             }
 
         });
+        
+        return new Page(lJobs, totalNbJobs);
+    }
+    
+
+    public Page<TaskState> getTaskStates(final long from, final long to, final String tag,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)